RevenueCat/Scripts/Purchases.cs

```diff
@@ -164,6 +167,14 @@ public void SetDebugLogsEnabled(bool logsEnabled)
         _wrapper.SetDebugLogsEnabled(logsEnabled);
     }
 
+    // ReSharper disable once UnusedMember.Global
+    public void SetProxyURL(string proxyURL)
+    {
+        _wrapper.SetProxyURL(proxyURL);
+        Setup(string.IsNullOrEmpty(appUserID) ? null : appUserID);
+        GetProducts(productIdentifiers, null);
```

Comment:
this bit is up for discussion. 
On other SDKs, we would ask devs to call `SetProxyURL` before setup. but here, setup isn't dev-controlled, it happens automatically from the params passed into the GUI. `GetProducts` is also called automatically. 
Since if you're using a proxy, you'd miss those calls, I'm making them here. 

An alternative approach would have been for this to be an extra param in the GUI, like `userDefaultsSuiteName`. However, I think we didn't want to expose that value too much, since it's really only meant for very specific usage. 

Come to think about it, this does mean, though, that your app may actually perform a request to the revenueCat servers before hitting the proxy. 
So if we want to be absolutely sure that our servers are never hit directly, it'll have to be a param in the GUI. 

Thoughts?

Reply:
Is there a way in Unity to add like a dropdown to the GUI? So that we can put this under an `Advanced` option under the `userDefaultsSuiteName` in the GUI? I have no clue. I will check.

Reply:
Ok, I checked and there's no way to do like a dropdown type of thing that hides some Advanced attributes, but we could add a header that says Advanced
https://unity3d.college/2017/05/22/unity-attributes/

Reply:
great idea! I'm adding it, will test and update the PR
<img width="358" alt="image" src="https://user-images.githubusercontent.com/3922667/87057483-4eab3e00-c1dd-11ea-9db0-0f0e889091f2.png">

Reply:
tested and working 🎉